src/observers/docsLoggerObserver.ts

```diff
@@ -94,6 +135,82 @@ export class DocsLoggerObserver {
         this.appendLine(`${tag}${event.message}`);
     }
 
+    // Build
+    private handleRepositoryInfoRetrieved(event: RepositoryInfoRetrieved) {
+        let repositoryUrl: string;
+        if (event.sourceRepositoryUrl !== event.repositoryUrl) {
+            repositoryUrl = `${event.sourceRepositoryUrl}(original)`;
+        } else {
+            repositoryUrl = event.repositoryUrl;
+        }
+        this.appendLine(`Repository Information of current workspace folder:`);
+        this.appendLine(`  Repository URL: ${repositoryUrl}`);

```

Comment:
>Repository URL: ${repositoryUrl}`); [](start = 27, length = 35)

It's a bit confused seems to me. Can we ask PM to provide some ideas?

Reply:
refine to:
```
Repository Information of current workspace folder:
  Repository URL: [https://github.com/928PJY/edge-developer(original](https://github.com/928PJY/edge-developer(original): [https://github.com/MicrosoftDocs/edge-developer](https://github.com/MicrosoftDocs/edge-developer))
  Repository Branch: master
```
would that be better?

---
In reply to: [365656073](https://github.com/docascode/vscode-docs-build/pull/13#discussion_r365656073) [](ancestors = 365656073)